src/.clang-format

```diff
@@ -15,7 +15,7 @@ BreakBeforeBinaryOperators: false
 BreakBeforeBraces: Allman
 BreakBeforeTernaryOperators: false
 BreakConstructorInitializersBeforeComma: false
-ColumnLimit:     0
+ColumnLimit:     120
```

Comment:
I think that we decided in slack on 120 columns, not "80 columns unless you really need it".  I don't really want to reopen this discussion.